docs/docs/cmd/teams/meeting/meeting-create.mdx

```diff
@@ -0,0 +1,241 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams meeting create
+
+Create a new online meeting
+
+## Usage
+
+```sh
+m365 teams meeting create [options]
+```
+
+## Options
+
+```md definition-list
+`-s --startTime [startTime]`
+: The start time of the meeting. If not specified, the startTime will be set to the current time.
+
+`-e --endTime [endTime]`
+: The end time of the meeting. If not specified, the endTime will be set to one hour after the startTime.
+
+`--subject [subject]`
+: The subject of the meeting.
+
+`-p, --participants [participants]`
+: A comma-separated list of participant UPNs.
+
+`--organizerEmail [organizerEmail]`
+: The organizer's email address. Requires application permissions.
+
+`-r, --recordAutomatically`
+: When using this flag, the meeting will be recorded automatically.
+```
+
+<Global />
+
+## Examples
+
+Create a new online meeting for the currently logged-in user, starting immediately and ending after one hour. 
+
+```sh
+m365 teams meeting create
+```
+
+Create a new online meeting for the currently logged-in user, with a specified start date and a duration of one hour.
+
+```sh
+m365 teams meeting create --startTime "2023-09-21T13:30:00Z"
+```
+
+Create a new online meeting for the currently logged-in user, with specified start and end dates.
+
+```sh
+m365 teams meeting create --startTime "2023-09-21T13:30:00Z" --endTime "2023-09-21T23:55:00Z"
+```
+
+Create a new online meeting for the currently logged-in user, with a specified subject.
+
+```sh
+m365 teams meeting create --startTime "2023-09-21T13:30:00Z" --endTime "2023-09-21T23:55:00Z" --subject "Test Subject"
+```
+
+Create a new online meeting for the currently logged-in user, with a specified subject and a list of participants.
+
+```sh
+m365 teams meeting create --subject "Test Subject" --participantEmails "john.doe@contoso.com,olga.manager@contoso.com"
```

Comment:
Let's change this to participantUserNames

Reply:
Done